test/helpers/HypTokenTest.sol

```diff
@@ -34,6 +34,7 @@ abstract contract HypTokenTest is Test {
 
     // origin chain
     // ---------------------------
+    // solhint-disable-next-line var-name-mixedcase
```

Comment:
Nice 🙏🏻

Reply:
Can we have a way for the Solidity linter warnings to show in each PRs in the file changes?